test/config/integration/certs/expired_cert_hash.h

```diff
@@ -0,0 +1,3 @@
+// NOLINT(namespace-envoy)
+constexpr char TEST_EXPIRED__CERT_HASH[] = "FC:F7:07:14:C3:0D:B4:BE:0B:BF:23:9B:C2:09:DA:CD:54:66:"
+                                           "32:65:07:50:35:E8:D0:14:ED:D6:B1:96:A1:3C";
```

Comment:
I think this is unused. Delete?

Reply:
Now `certs.sh` will generate a hash file, not only here, but also in other tests, although some of the hash files are unused.

Reply:
OK I see, that's fine, thanks.